msteams-platform/task-modules-and-cards/cards/cards-format.md

```diff
@@ -1022,6 +1024,353 @@ When a user selects the overflow menu on mobile, Adaptive Card displays the butt
 
 ---
 
+## Borders in Adaptive Cards
+
+Adaptive Cards support a wide variety of elements, but having too many elements clutters the card and hinders readability. You can add borders to various elements in an Adaptive Card to delineate them, making it easier for users to distinguish between them.
```

Comment:
I think the doc is fine overall, but I do feel it's overkill to have two distinct sections for borders and rounded corners. I think it would be better to condensate into a single topic "Borders and rounded corners" and only have one sample that showcases both.

Reply:
I agree, @dclaux. We could condense it into one section with a single snippet and it would make for a simpler document and reduce the length of the article (which is much needed). However, here's why I kept it separate.

- Borders and rounded corners are two different features that serve distinct purposes. My understanding is that borders are used for delineation and rounded corners are more geared towards aesthetics.
- SEO-wise, having them under two different second-level headings helps with optimization and discoverability.
- There is a slim possibility that some developers might think that these two features must be implemented together if they're under the same heading.

Let me know what you think. I am on board both ways.

Reply:
I agree, @dclaux. I've combined both sections. Please take a look and let me know if it looks good.